zingolib/src/wallet/describe.rs

```diff
@@ -302,6 +302,14 @@ impl LightWallet {
             .map(|(_index, sk)| *sk)
             .collect::<Vec<_>>()
     }
+
+    /// gets the basic receiver for the wallet. this is the only receiver implemented as 2024-09-22
```

Comment:
What does "basic receiver" mean?

Note that this will always return the first element of the iterator. One could assume that it has some kind of order, but it may not be the case, so the returned address might not be the one intended.

Reply:
thanks for the review! I will adjust.

Reply:
This is resolved, in the future let's say Option<foo> or optionally returns (instead of Some(foo) and None).